src/backend/core/tests/entitlements/test_backends.py

```
@@ -0,0 +1,212 @@
+
+    @responses.activate
+    def test_get_user_entitlements_no_access_token(self):
+        responses.add(
+            responses.GET,
+            "https://deploycenter.example.com/api/v1.0/entitlements",
+            json={"entitlements": {"can_access": True, "can_admin_maildomains": []}},
+            status=200,
+        )
+
+        backend = self._get_backend()
+        result = backend.get_user_entitlements("user-sub", "user@example.com")
```

Ruff (0.15.1)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)